File: CS/Dashboard_Wpf_Localizer_Example/MyDashboardLocalizer.cs
using DevExpress.DashboardCommon.Localization;
using System;

namespace Dashboard_Wpf_Localizer_Example
{
    public class MyDashboardLocalizer : DashboardLocalizer
    {
        public override string GetLocalizedString(DashboardStringId id)
        {
            return Enum.GetName(typeof(DashboardStringId), id);
        }
    }
}

File: VB/Dashboard_Wpf_Localizer_Example/MyDashboardLocalizer.vb
Imports DevExpress.DashboardCommon.Localization
Imports System

Namespace Dashboard_Wpf_Localizer_Example
	Public Class MyDashboardLocalizer
		Inherits DashboardLocalizer

		Public Overrides Function GetLocalizedString(ByVal id As DashboardStringId) As String
			Return System.Enum.GetName(GetType(DashboardStringId), id)
		End Function
	End Class
End Namespace